data/styles/clear/include/Basemap.mapcss

```diff
@@ -238,6 +236,11 @@ area|z15-[natural=water][tunnel],
 area|z16-[amenity=fountain],
 {fill-color: @water;}
 
+area|z14-[natural=water][water=ditch],
+area|z14-[natural=water][water=drain],
+area|z14-[natural=water][water=wastewater],
+{fill-color: @water_bad;}
```

Comment:
This way it'll make the color change between -z13 and z14-.
So either change it all the way or remove area fills for z1-z13 (makes sense!)

Reply:
I've changed it so the visibility roughly matches normal water types 👍